core/frontend/src/App.vue

```diff
@@ -719,6 +734,12 @@ export default Vue.extend({
         this.$router.push('/')
       }
     },
+    buildDateClick(): void {
+      this.build_clicks = (this.build_clicks + 1) % 10
+      if (this.build_clicks === 0) {
+        settings.is_dev_mode = true
+      }
```

Comment:
Is there a reason for not doing something simpler ?
```suggestion
      this.build_clicks += 1
      settings.is_dev_mode = this.build_clicks > 10
```

Reply:
Just to keep consistent that if the user activate and deactivate the dev mode without refreshing it still will require 10 clicks and not only one in the second activation.